src/endpoint-configuration-service/src/main/java/org/fiware/sidecar/exception/CredentialsConfigNotFound.java

```diff
@@ -0,0 +1,25 @@
+package org.fiware.sidecar.exception;
+
+import lombok.Getter;
+
+import java.util.List;
+
+public class CredentialsConfigNotFound extends Exception {
```

Comment:
Missing JavaDoc describing use of each class.

Reply:
Fixed in a couple of commits